### PR TITLE
OPSEXP-2151 Refactor RPMs build

### PR DESCRIPTION
**Pull request by gionn**
Ref: OPSEXP-2151

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -2,134 +2,78 @@ name: Build
...
+        run: >-
+          rpm -qp --requires ImageMagick-libs-${{ steps.variables.outputs.imagemagick_version }}.${{ matrix.target_arch }}.rpm |
+          grep -qEv 'libcdt|libcgraph|libgvc|libgs|libMagickCore|libMagickWand'
+
       - uses: actions/upload-artifact@v3
         with:
-          name: centos-rpms
-          path: rpms/*.rpm
+          name: ${{ matrix.nexus_classifier }}-rpm
+          path: RPMS/${{ matrix.target_arch }}/*.rpm
```
Comment (alxgomz) [inline .github/workflows/build.yml:57]: Any reason why we're uploading these artifacts? I don't see them being downloaded anywhere

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -164,10 +107,10 @@ jobs:
 
       - name: Test
         run: "dpkg-deb -f imagemagick-alfresco_${{ steps.variables.outputs.imagemagick_version }}_amd64.deb Depends | grep -qEv 'libcdt|libcgraph|libgvc|libgs9'"
-      
+
       - uses: actions/upload-artifact@v3
```
Comment (alxgomz) [inline .github/workflows/build.yml:111]: same question

File: .github/workflows/build.yml (hunk 1 above)
Comment (gionn) [reply .github/workflows/build.yml:57]: how do you test manually those packages before releasing them on nexus?

**Review by alxgomz — COMMENTED** [review]
If that's the sole purpose I think it would be much better to test during build:
* test installing on clean image
* test running convert on a sample file
Do you see any pbm doig that (as I'm writing it that sounds like rather quick)?

**Comment by gionn** [comment]
> Do you see any pbm doing that (as I'm writing it that sounds like rather quick)?

In a new task OPSEXP-2224

**Review by alxgomz — APPROVED** [review]

**Pull request merged**